src/main/scala/com/fortysevendeg/android/sqlite/metadata/SQLiteDatabaseMetaDataSupport.scala

```diff
@@ -0,0 +1,210 @@
+
+  override def supportsAlterTableWithDropColumn(): Boolean = false
+
+  override def supportsANSI92EntryLevelSQL(): Boolean = false
+
+  override def supportsANSI92FullSQL(): Boolean = false
+
+  override def supportsANSI92IntermediateSQL(): Boolean = false
+
+  override def supportsBatchUpdates(): Boolean = true
+
+  override def supportsCatalogsInDataManipulation(): Boolean = false
+
+  override def supportsCatalogsInIndexDefinitions(): Boolean = false
+
+  override def supportsCatalogsInPrivilegeDefinitions(): Boolean = false
+
+  override def supportsCatalogsInProcedureCalls(): Boolean = false
+
+  override def supportsCatalogsInTableDefinitions(): Boolean = false
+
+  override def supportsColumnAliasing(): Boolean = true
+
+  override def supportsConvert(): Boolean = false
+
+  override def supportsConvert(fromType: Int, toType: Int): Boolean = false
+
+  override def supportsCoreSQLGrammar(): Boolean = true
+
+  override def supportsCorrelatedSubqueries(): Boolean = false
+
+  override def supportsDataDefinitionAndDataManipulationTransactions(): Boolean = true
+
+  override def supportsDataManipulationTransactionsOnly(): Boolean = false
+
+  override def supportsDifferentTableCorrelationNames(): Boolean = false
+
+  override def supportsExpressionsInOrderBy(): Boolean = true
+
+  override def supportsExtendedSQLGrammar(): Boolean = false
+
+  override def supportsFullOuterJoins(): Boolean = false
+
+  override def supportsGetGeneratedKeys(): Boolean = true
+
+  override def supportsGroupBy(): Boolean = true
+
+  override def supportsGroupByBeyondSelect(): Boolean = false
+
+  override def supportsGroupByUnrelated(): Boolean = false
+
+  override def supportsIntegrityEnhancementFacility(): Boolean = false
+
+  override def supportsLikeEscapeClause(): Boolean = false
+
+  override def supportsLimitedOuterJoins(): Boolean = true
+
+  override def supportsMinimumSQLGrammar(): Boolean = true
+
+  override def supportsMixedCaseIdentifiers(): Boolean = true
+
+  override def supportsMixedCaseQuotedIdentifiers(): Boolean = false
+
+  override def supportsMultipleOpenResults(): Boolean = false
+
+  override def supportsMultipleResultSets(): Boolean = false
+
+  override def supportsMultipleTransactions(): Boolean = true
+
+  override def supportsNamedParameters(): Boolean = true
+
+  override def supportsNonNullableColumns(): Boolean = true
+
+  override def supportsOpenCursorsAcrossCommit(): Boolean = false
+
+  override def supportsOpenCursorsAcrossRollback(): Boolean = false
+
+  override def supportsOpenStatementsAcrossCommit(): Boolean = false
+
+  override def supportsOpenStatementsAcrossRollback(): Boolean = false
+
+  override def supportsOrderByUnrelated(): Boolean = false
+
+  override def supportsOuterJoins(): Boolean = true
+
+  override def supportsPositionedDelete(): Boolean = false
+
+  override def supportsPositionedUpdate(): Boolean = false
+
+  override def supportsResultSetConcurrency(`type`: Int, concurrency: Int): Boolean =
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [ type  is a reserved keyword don't use it as a name.](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/784801319/issues/source?bid=2442814&fileBranchId=2493741#l154)